tests/xharness/Jenkins/Jenkins.cs

```diff
@@ -15,6 +15,36 @@
 using Xharness.Jenkins.Reports;
 
 namespace Xharness.Jenkins {
+	
+	[Flags]
+	public enum TestSelection : long {
+		MacOS = 0,
+		iOS32 = 1,
+		iOS64 = 2,
+		iOS = iOS32 | iOS64,
+		iOSExtensions = 8,
+		iOSMSBuild = 16,
+		tvOS = 32,
+		watchOS = 64,
+		Mmp = 128,
+		Mtouch = 256,
+		Btouch = 512,
+		MacBindingProject = 1024,
+		Simulator = 2048,
+		OldiOSSimulator = 4096,
+		Device = 8192,
+		Xtro = 16384,
+		Cecil = 32768,
+		Docs = 65536,
+		BclXUnit = 131072,
+		BclNUnit = 262144,
+		Bcl = BclXUnit | BclNUnit,
+		Mscorlib = 524288,
+		NonMonotouch = 1048576,
+		Monotouch = 2097152,
+		DotNet = 4194304,
```

Comment:
Maybe use bit shifts instead, that way we don't have to use higher math to find the next value:

```csharp
MacOS = 1 << 0,
iOS32 = 1 << 1,
iOS64 = 1 << 2,
iOSExtensions = 1 << 3,
...
```